src/split-wrapper.js

```diff
@@ -0,0 +1,74 @@
+// split-wrapper.js
+var _ = require('underscore');
+
+var createLm = require('./create-lm');
+var createState = require('./create-state');
+var wrapper = require('./javascript-wrapper');
+var wrapperHelper = require('./wrapper-helper');
+
+/**
+ *  Define a default updater stub function to be used if the caller
+ * did not pass one in.  Simply returns whatever the original input was so it
+ * can be sent through the output port to the next component
+ *
+ * @param vnid_hash hash of vnids and data values 
+ */
+function defaultUpdater(vnid_hash) {
```

Comment:
I put it in as a place holder. Although it looks similar here (one parameter, returned), we are working with a hash here.  For this version, however, we certainly could use the default javascript wrapper updater.  Join is another matter...